### ISSUE TITLE
How do I delete a sale?

### ISSUE THREAD

**Issue by josefnpat** [itchio/itch.io#343]
![image](https://cloud.githubusercontent.com/assets/646162/12603490/db202c0e-c478-11e5-94c0-6010677baa2b.png)

**Comment by leafo** [comment]
you don't :smiling_imp:

**Comment by josefnpat** [comment]
Hmmm....

I will tell you my workflow so we can figure out a workaround maybe?

I was trying to figure out if I wanted a "sale" or a "bundle". So I made a "sale" and a "bundle" and found I wanted the bundle. I was able to delete the bundle, but not the sale.

Perhaps don't allow sales that have been published to be deleted? Or do a soft delete?

**Comment by leafo** [comment]
yup, I agree it should be possible to remove a sale. It can only be set to draft mode right now

**Comment by josefnpat** [comment]
Gotcha; feel free to triage this ticket as you wish then!
